WebJobs.Extensions.Sql.sln

```diff
@@ -7,7 +7,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SqlExtensionSamples", "samp
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SqlBinding", "src\SqlBinding\SqlBinding.csproj", "{4453B407-2CA3-4011-BDE5-247FCE5C1974}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SqlExtension.Tests", "test\SQLExtension.tests\SqlExtension.Tests.csproj", "{A5B55530-71C8-4A9A-92AD-1D33A5E80FC1}"
```

Comment:
nit: rather than changing the csproj to match the folder name & ending up with weird casing, we should fix the folder name (SqlExtension.Tests)

Reply:
Changed paths and files to all be "SqlExtension.Tests"